deployment/docker/Makefile

```diff
@@ -173,15 +176,22 @@ package-via-docker-386:
 remove-docker-images:
 	docker image ls --format '{{.Repository}}\t{{.ID}}' | awk '{print $$2}' | xargs docker image rm -f
 
-
 docker-single-up:
-	docker-compose -f deployment/docker/docker-compose.yml up
+	rm -rf deployment/docker/grafana-plugins/victoriametrics-datasource
+	wget -nc https://github.com/VictoriaMetrics/grafana-datasource/releases/download/v$(VM_DATASOURCE_VERSION)/victoriametrics-datasource-v$(VM_DATASOURCE_VERSION).tar.gz \
+	-O deployment/docker/grafana-plugins/victoriametrics-datasource-v$(VM_DATASOURCE_VERSION).tar.gz
+	tar -xzvf deployment/docker/grafana-plugins/victoriametrics-datasource-v$(VM_DATASOURCE_VERSION).tar.gz -C deployment/docker/grafana-plugins/
```

Comment:
there are following issues with this change:
* it always re-downloads the plugin even if it already exists locally
* it repeats itself in the `docker-cluster-up` command

I suggest to move the download process to a separate Makefile command and do the downloading only if datasource doesn't exist already. Would be great to have an ability to automatically re-download dataource if version `VM_DATASOURCE_VERSION ` doesn't match existing files.

Reply:
fixed